README.md

```diff
@@ -36,15 +36,15 @@ application's specific functionality and prototyping PoCs.
 Installation
 ============
 
-You can install FakeNet-NG in a few different ways.
+You can install FakeNet-NG in a few different ways. Note that the following installation processes will retrieve third-party open-source libraries used by FakeNet-NG to your system. These libraries will be dynamically loaded at runtime, and some of these libraries may be GPL or LGPL licensed.
```

Comment:
I think we should be explicit about the licenses of dependencies if we're going to call them out generally here. maybe add a list of deps and their license and orgs below?